libraries/postgres/src/Database.js

```diff
@@ -386,6 +386,18 @@ class Database {
     assert.deepEqual(current, schema.tables.get());
   }
 
+  async _checkVersion({db, schema}) {
+    await db._withClient('admin', async client => {
+      const version = await client.query(`
+        SELECT current_setting('server_version_num');
+      `);
+      const ver = version.rows[0].current_setting;
+      if (ver < 110000) {
+        throw new Error("Postgres version is less than 11. Please upgrade to 11.x or above.");
+      }
```

Comment:
I think we'll want to check for and prohibit >= 120000 as well.  At the moment, we don't test on such a version so we do not want to run it there.

Reply:
Fixed